source/wrapper/test/Wrapper-unit.js

```diff
@@ -329,4 +331,68 @@ contract('Wrapper Unit Tests', async accounts => {
       )
     })
   })
+
+  describe('Test pausability', async () => {
+    it('Test setPausedStatus is only callable by owner', async () => {
+      await reverted(wrapper.setPausedStatus(true, { from: notOwner }))
+    })
+
+    it('Test swap when paused', async () => {
+      let notWethContract = mockFT.address
+
+      const order = await orders.getOrder({
+        signer: {
+          token: notWethContract,
+          wallet: mockSigner,
+        },
+        sender: {
+          wallet: mockSender,
+          param: senderParam,
+          token: notWethContract,
+        },
+      })
+
+      await wrapper.setPausedStatus(true)
+
+      await reverted(
+        wrapper.swap(order, {
+          value: 0,
+          from: mockSender,
+        }),
+        'CONTRACT_IS_PAUSED'
+      )
+    })
+
+    it('Test swap when not paused', async () => {
+      let notWethContract = mockFT.address
+
+      const order = await orders.getOrder({
+        signer: {
+          token: notWethContract,
+          wallet: mockSigner,
+        },
+        sender: {
+          wallet: mockSender,
+          param: senderParam,
+          token: notWethContract,
+        },
+      })
+
+      await passes(
+        wrapper.swap(order, {
+          value: 0,
+          from: mockSender,
+        })
+      )
+    })
+
+    it('Test self destruct when paused', async () => {
+      await wrapper.setPausedStatus(true)
+      await passes(wrapper.killContract(owner))
```

Comment:
You can actually check a self destruct more thoroughly as follows.
```
await wrapper.killContract(owner, { from: owner })

let contractCode = await web3.eth.getCode(wrapper.address)
equal(contractCode, '0x', 'contract did not self destruct')
```

Reply:
The way self destruct works is literally removing the contract code from existence at the address - so you can check that its gone